safe-report-comments.php

```diff
@@ -261,132 +275,149 @@ public function mark_comment_moderated( $comment ) {
 				update_comment_meta( $comment->comment_ID, $this->_plugin_prefix . '_moderated', true );
 			}
 		}
-		
+
 		/*
 		 * Check if this comment was flagged by the user before
 		 */
-		public function already_flagged( $comment_id ) {		
+		public function already_flagged( $comment_id ) {
 
 			// check if cookies are enabled and use cookie store
-			if( isset( $_COOKIE[ TEST_COOKIE ] ) ) {
+			if ( isset( $_COOKIE[ TEST_COOKIE ] ) ) {
 				if ( isset( $_COOKIE[ $this->_storagecookie ] ) ) {
 					$data = $this->unserialize_cookie( $_COOKIE[ $this->_storagecookie ] );
 					if ( is_array( $data ) && isset( $data[ $comment_id ] ) ) {
 						return true;
 					}
 				}
-			} 
-			
-			
+			}
+
+			// phpcs:ignore WordPressVIPMinimum.Variables.ServerVariables.UserControlledHeaders
+			$remote_addr = filter_var( isset( $_SERVER['REMOTE_ADDR'] ) ? $_SERVER['REMOTE_ADDR'] : '', 'FILTER_VALIDATE_IP' );
```

Comment:
Consider changing to use `filter_input()`, which should then move away from needing the PHPCS ignore as well.